packages/vuetify/src/composables/border-radius.ts

```diff
@@ -37,7 +39,7 @@ export const makeBorderRadiusProps = propsFactory({
 export function useBorderRadius (props: BorderRadiusProps) {
   const borderRadiusClasses = computed(() => {
     const classes = []
-    const rounded = props.rounded
+    const rounded = props && 'rounded' in props ? props.rounded : null
```

Comment:
There was this missing possibility

Reply:
What was? If `'rounded' in props` is false then `props.rounded` is undefined and the `rounded == null` condition matches.

Reply:
Fair enough, adjusted